assets/app/scripts/services/data.js

```diff
@@ -1019,8 +1066,15 @@ angular.module('openshiftConsole')
       namespace: namespace,
       q: params
     };
+    // TODO (bpeterse): The URL_OBJECT_SUBRESOURCE & URL_NAMESPACED_OBJECT_SUBRESOURCE
+    // have both been removed.  Need to figure out what to use now in place of these.
     if (isWebsocket) {
-      template = namespaceInPath ? URL_NAMESPACED_WATCH_LIST : URL_WATCH_LIST;
+      if (subresource) {
+        templateOptions.subresource = subresource;
+        template = namespaceInPath ? URL_NAMESPACED_OBJECT : URL_OBJECT;
+      } else {
+        template = namespaceInPath ? URL_NAMESPACED_WATCH_LIST : URL_WATCH_LIST;
+      }
```

Comment:
websocket == ws/wss, otherwise http/https
watch is now just a param on a list API call, so handle it generically like any other param